setup.cfg

```diff
@@ -8,20 +9,53 @@ project_urls =
     Documentation = https://btrack.readthedocs.io/en/stable/
     Source Code = https://github.com/quantumjot/BayesianTracker
     User Support = https://github.com/quantumjot/BayesianTracker/issues
-description-file = README.md
+description = A framework for Bayesian multi-object tracking
+long_description = file: README.md
 classifier =
-  Operating System :: OS Independent
   Programming Language :: C++
   Programming Language :: Python
   Programming Language :: Python :: 3 :: Only
   Topic :: Scientific/Engineering
   Topic :: Scientific/Engineering :: Bio-Informatics
   Topic :: Scientific/Engineering :: Image Recognition
   Framework :: napari
+  Operating System :: Microsoft :: Windows
+  Operating System :: POSIX
+  Operating System :: Unix
+  Operating System :: MacOS
+
+[options]
+packages = find:
+python_requires = >=3.8
+include_package_data = True
+install_requires =
+  cvxopt>=1.2.0
+  h5py>=2.10.0
+  numpy>=1.17.3
+  pooch>=1.0.0
+  pydantic>=1.9.0
+  scikit-image>=0.16.2
+  scipy>=1.3.1
+
+[options.package_data]
+btrack =
+  napari.yaml
+  libs/libtracker*
+  VERSION.txt
+
+[options.extras_require]
```

Comment:
I'd move this up so it's directly under the `install_requires` section.

Reply:
Done!